docs/changes/table/12930.bugfix.rst

```diff
@@ -0,0 +1 @@
+Switch ``Table`` connector for ASDF to asdf-astropy to prevent deprecation warnings.
```

Comment:
@saimn , @Cadair , or @bsipocz , is there a way to merge entries from two different PRs together like back in the good old days? This PR is a continuation of another unreleased PR at #12903 .

Reply:
@pllim - I believe you need to copy the very same fragment file with the filename of the new PR. At least that is how it used to work (towncrier's docs is not very detailed, so I'm not sure it still works that way).

Example is here, grep for the same content line for a duplicate below, and the gist file for the rendered output in the PR description:
https://github.com/astropy/astropy/pull/11279/files#diff-cda4a9d39c6ec7ba558bfb35e0473957b5d223431d36dc0a63f3544d27a878f5R1

Reply:
I replaced this file with one that has identical content to 12903